nengo_dl/tests/test_benchmarks.py

```diff
@@ -196,10 +196,10 @@ def test_lmu(Simulator, native_nengo, pytestconfig):
 @pytest.mark.parametrize(
     "net, train, minibatch_size, min, max",
     [
-        (benchmarks.cconv(128, 64, nengo.RectifiedLinear()), False, 64, 0.6, 0.75),
-        (benchmarks.cconv(128, 64, nengo.LIF()), False, 64, 1.4, 1.6),
-        (benchmarks.integrator(128, 32, nengo.RectifiedLinear()), True, 64, 0.55, 0.9),
-        (benchmarks.integrator(128, 32, nengo.LIF()), True, 64, 0.95, 1.25),
+        (benchmarks.cconv(128, 64, nengo.RectifiedLinear()), False, 64, 0.95, 1.1),
```

Comment:
Would it be possible to keep the benchmarks from `abrgpu` around somewhere? It might be nice to have a command-line pytest option that specifies where we're running, and chooses the appropriate numbers. When we do a release, we might want to run the benchmarks on both `abrgpu` and Azure just to make sure they haven't regressed in either location.

Reply:
To be honest, I suspect we wouldn't get a lot out of occasionally running things on `abrgpu`. I'm just picturing if we did detect a regression on `abrgpu` that hadn't shown up on Azure, I think 99% of the time we would just chalk it up to environmental differences and either ignore it or spend a bunch of time fixing the `abrgpu` environment (rather than touching any code in Nengo DL). We'd also need to maintain both sets of benchmarks as we go, which is a pain (whenever we need to adjust these benchmarks it sucks up a bunch of time rerunning the builds over and over and tweaking the numbers).

So, long story short, it'd be great to have a more robust regression testing framework that was regularly testing against a bunch of different environments. But I wouldn't dedicate the time to doing a partial version of that now given our resources.

Reply:
That makes sense. My main concern is that the GPUs on Azure are pretty crappy, and not that representative of what we'll typically be running on. But you're right that the regression tests aren't great themselves, and perhaps also not fully representative of a larger end-to-end training, which is where we more care about performance.